### PR TITLE
Grit/life long learning

### PR DESCRIPTION
**Pull request by laderast**
Added discussion of grit and lifelong learning to episode 9. Added as many references as I could.

### CODE REVIEW

**Comment by ChristinaLK** [comment]
I love this!  Thanks @laderast 
If you're still making changes, you could add a reference to live coding in the error framing section as a method that creates space for errors+error framing as a teaching tool, but I don't think that's necessary to merge.

**Comment by laderast** [comment]
@ChristinaLK Thanks! I will try and add a mention of live coding.

**Comment by laderast** [comment]
@ErinBecker Thanks! Glad you liked it.

File: _episodes/09-mindset.md [hunk 1]
```diff
@@ -47,44 +47,65 @@ strategies that will help you promoting a growth mindset in your learners and yo
...
+- *Promoting Perseverance*. Part of grit is being able to push through even when the going is difficult. Improvement-based praise is an important motivating tool to promote perseverance.
+- *Sharing Passion*. Talking about why you persist in working in your field despite the difficulties and challenges is another way to instill grit.
+- *Emphasizing Long-Term Goals*. Showing the effects of learning the concepts in terms of student's individual long-term goals can instill passion and drive for learning despite the difficulties of the topic.
+
 ### Becoming a lifelong learner  
 
+Surviving the job market nowadays requires the ability to be a lifelong learner, in order to learn and adapt to new skills that may be required by employers. Lifelong learners are capable of what Cal Newport calls "[deep work](http://calnewport.com/books/deep-work/)": defined as the "ability to focus intensely on cognitively demanding tasks".
```
Comment (lexnederbragt) [inline _episodes/09-mindset.md]: I have not read Cal Newport's book, but am following his blog. Based on that, I am not certain that he actually says that "Lifelong learners are capable of deep work" - but I'm gladly proven wrong.

**Comment by lexnederbragt** [comment]
Great addition! I am going to discuss this with the TA's of the course I'm teaching this fall.

**Comment by laderast** [comment]
Hi Lex,

Newport references deep work as an essential skill for knowledge work, which requires a lifelong learning mindset. The connection is not explicit. I'm happy to change this if you feel it better reflects his view.

Ted

**Comment by lexnederbragt** [comment]
Thanks - please do so we are not misattributing something to (the excellent) Cal Newport (I'm probably being over-careful, but still...).

**Pull request merged**